docs/sphinx/source/clearsky.rst

```diff
@@ -118,6 +114,57 @@ The Ineichen and Perez clear sky model parameterizes irradiance in terms
 of the Linke turbidity [Ine02]_. pvlib-python implements this model in
 the :py:func:`pvlib.clearsky.ineichen` function.
 
+
+The :py:func:`~pvlib.atmosphere.kasten96_lt` function can be used to calculate
+Linke turbidity [Kas96]_ as input to the clear sky Ineichen and Perez function.
+The Kasten formulation requires precipitable water and broadband AOD which can
+be approximated at 700-nm [Mol98]_ or from the function,
+:py:func:`~pvlib.atmosphere.bird_hulstrom80_aod_bb` which uses a combination of
+AOD measured at two wavelengths [Bir80]_.
+
+.. ipython::
+
+    In [1]: MBARS = 100  # conversion factor from mbars to Pa
+
+    In [1]: URL = 'http://rredc.nrel.gov/solar/old_data/nsrdb/1991-2005/data/tmy3/722040TYA.CSV'
+
+    In [1]: melbourne = tmy.readtmy3(URL)
+
+    In [1]: dt = pd.DatetimeIndex(start='2000-01-01 01:00:00', end='2000-12-31', freq='H')
+
+    In [1]: melbourne[0].index = dt
```

Comment:
Suggest unpacking the tuple from `readtmy3` up front, to simplify later references to the content, and make the interface between this example and `readtmy3` more compact.

Reply:
I would say I agree, but I see this is marked as resolved so maybe I'm misunderstanding.